sound/soc/sof/mediatek/mt8195/mt8195.c

```diff
@@ -256,6 +350,25 @@ static int mt8195_dsp_probe(struct snd_sof_dev *sdev)
 		goto err_adsp_sram_power_off;
 	}
 
+	priv->ipc_dev = platform_device_register_data(&pdev->dev, "mtk-adsp-ipc",
+						      PLATFORM_DEVID_NONE,
+						      pdev, sizeof(*pdev));
```

Comment:
where is the platform_device_unregister() done?

Reply:
Sorry, I miss this part, will fix this. thanks.